### ISSUE TITLE
Scramble bots shown in tags.

### ISSUE THREAD

**Issue by Rotinx** [DiscordBotList/issues#376]
#### Suggestion

Have bots randomly scrambled within tag groups., but still have a option to have them displayed in voting order.

This makes it easier for bots to be discovered who have less votes. Though this feature should be limited to bots which are actually online.

#### Applicable Links or Attachments

https://top.gg/tag/*

**Comment by Luc1412** [comment]
I personally think this could be a good way to feature small Bots, but it shouldn't become the default sorting type.

Voting should still benefit the bot owner. The chance of getting in the first page of Top Votes is very low, but you still can get a good rank on your Tag.

I think the best implementation of such feature would be a Randomize Button or general sorting types like on popular listing sites. (Most Votes -> Lowest Votes, Lowest Votes -> Most Votes, New, Random etc.)

**Comment by Rotinx** [comment]
I agree @Luc1412 we still want to reward bots who gather the most votes, although making it easier to find newer/smaller bots.
